Pod/Classes/JNKeychain.h

```diff
@@ -13,6 +13,12 @@
 @interface JNKeychain : NSObject
 
 /**
+  @abstract Sets an access group identifier
```

Comment:
Could you clarify that once set, this value will be used for all further saved values?